modules/tinymce/src/themes/silver/main/ts/ui/dialog/Slider.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { AlloyTriggers, Behaviour, Focusing, SimpleSpec, Slider, SliderTypes } from '@ephox/alloy';
+import { Dialog } from '@ephox/bridge';
+import { Fun } from '@ephox/katamari';
+
+import { UiFactoryBackstageProviders } from '../../backstage/Backstage';
+import { ComposingConfigs } from '../alien/ComposingConfigs';
+import { formChangeEvent } from '../general/FormEvents';
+
+type SliderSpec = Omit<Dialog.Slider, 'type'>;
+
+export const renderSlider = (spec: SliderSpec, providerBackstage: UiFactoryBackstageProviders): SimpleSpec => {
+  const labelPart = Slider.parts.label({
+    dom: {
+      tag: 'label',
+      classes: [ 'tox-label' ],
+      innerHtml: providerBackstage.translate(spec.label)
```

Comment:
right, previously we had total control over the text because it was internal to the `imagetools` panel. I'll swap it to what our `label` component does, since that's basically what we're after anyway.

Reply:
Label does it the same way. I'll log a task.
https://github.com/tinymce/tinymce/blob/8ee937c891eabef49cb86cce605680b913915346/modules/tinymce/src/themes/silver/main/ts/ui/dialog/Label.ts#L22

Reply:
TINY-8340